CHANGELOG.md

```diff
@@ -8,6 +8,11 @@ document.
 
 [37f4c172...master](https://github.com/rust-lang/rust-clippy/compare/37f4c172...master)
 
+### New Lints
+
+* [`require_workspace_dependencies`]
+  [#11389](https://github.com/rust-lang/rust-clippy/pull/11389)
+
```

Comment:
The changelog is updated once per version, no need to update it per-commit (e.g. #11351). Leave the link at line 5307, but don't worry adding your changes to this section, someone will do it for you :heart: .